index.js

```diff
@@ -19,7 +19,7 @@ const assertPatternsInput = patterns => {
 };
 
 const checkCwdOption = options => {
-	if (options && options.cwd && !fs.statSync(options.cwd).isDirectory()) {
+	if (options && options.cwd && fs.existsSync(options.cwd) && !fs.statSync(options.cwd).isDirectory()) {
```

Comment:
This is wasteful. You're doing two fs calls. It could also cause a race issue where the file system might change between the calls. Instead, you can just try/catch `fs.statSync.`